pkg/block/block.go

```diff
@@ -116,13 +167,13 @@ var (
 	// StorageController::MarshalJSON.
 	// We use this table only in UnmarshalJSON, so it should be OK.
 	stringStorageController = map[string]StorageController{
-		"unknown": STORAGE_CONTROLLER_UNKNOWN,
-		"ide":     STORAGE_CONTROLLER_IDE,
-		"scsi":    STORAGE_CONTROLLER_SCSI,
-		"nvme":    STORAGE_CONTROLLER_NVME,
-		"virtio":  STORAGE_CONTROLLER_VIRTIO,
-		"mmc":     STORAGE_CONTROLLER_MMC,
-		"loop":    STORAGE_CONTROLLER_LOOP,
+		"unknown": StorageControllerUnknown,
+		"ide":     StorageControllerIDE,
+		"scsi":    StorageControllerSCSI,
+		"nvme":    StorageControllerNVMe,
+		"virtio":  StorageControllerVirtIO,
+		"mmc":     StorageControllerMMC,
+		"loop":    StorageControllerLoop,
```

Comment:
when we use reverse table like this I'm not sure the backward compatibility is ensured, but I think this should not be a bother. We can break stuff moving to 1.0.0 (and then don't for a long long while)

Reply:
ok, I misread that part. So we're golden!